.github/workflows/build-and-test.yml

```diff
@@ -156,7 +156,7 @@ jobs:
   unittest-matrix:
     strategy:
       matrix:
-        go-version: [1.18, 1.17]
+        go-version: [1.19, 1.18, 1.17]
```

Comment:
Drop 1.17? We say that we support only last 2?

Reply:
Yup, I was planning on turning up tests for 1.19 first to see what would break 😅